packages/webapp/pages/account/notifications.tsx

```diff
@@ -96,7 +96,12 @@ const AccountNotificationsPage = (): ReactElement => {
   const personalizedDigestType =
     personalizedDigest?.flags?.sendType || (!isLoading ? SendType.Off : null);
 
-  const { acceptedMarketing, notificationEmail } = user ?? {};
+  const {
+    acceptedMarketing,
+    notificationEmail,
+    followingEmail,
+    followNotifications,
+  } = user ?? {};
```

Comment:
Is this user object coming from boot or separate user query?

Reply:
From boot

Reply:
ok, let's keep it like that on API side as well, we will refactor when new notifications settings arrive